lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -718,7 +718,7 @@ function dosomething_helpers_get_current_user($user = NULL, $asArray = FALSE) {
   }
 
   // Authenticated User
-  $northstar_user = dosomething_northstar_get_user($user->uid);
+  $northstar_user = dosomething_northstar_get_user($user->uid, 'drupal_id');
```

Comment:
`drupal_id` is already the default in that function ;) unless there's an update to come which changes this...